package.json

```diff
@@ -40,7 +40,7 @@
     "simple-glob": "^0.2.0",
     "socket.io": "~2.2.0",
     "socket.io-client": "~2.2.0",
-    "sproxydclient": "scality/sproxydclient#45090b7",
+    "sproxydclient": "github:scality/sproxydclient#a6ec980",
```

Comment:
I've seen this prefix `github:` added when running `npm i`. It sounds harmless, but wondering if there's a reason we seem to consistently removing it from our package.json.

Reply:
It's npm inconsistency...some versions add, some don't.

Reply:
As long as we're compatible LGTM.